data/Templates/Hl7v2/Resource/_Patient.liquid

```diff
@@ -53,9 +55,11 @@
         "gender":"{{ PID.8.Value | get_property: 'CodeSystem/Gender', 'code' }}",
         "address":
         [
+            {% for p in PID.11.Repeats -%}
             {
-                {% include 'DataType/XAD' XAD: PID.11 -%}
+                {% include 'DataType/XAD' XAD: p -%}
```

Comment:
We need to make same changes in template generation tool.

Reply:
Yes, this feature has been added in the generation tool.

Reply:
By offline discussion, I'll submit the pull request in the generation tool.